sources/packages/web/src/components/common/IdleTimeChecker.vue

```diff
@@ -6,19 +6,31 @@
     @keyup="setLastActivityTime"
   >
     <slot></slot>
-    <ConfirmExtendTime ref="extendTimeModal" :clientIdType="clientIdType" />
+    <confirm-extend-time
+      ref="extendTimeModal"
+      :clientIdType="clientIdType"
```

Comment:
i think we can remove `:clientIdType="clientIdType"` as its no longer used in `confirm-extend-time`